math/gnudatalanguage/Portfile

```diff
@@ -23,18 +23,15 @@ description                 a free IDL combatible incremental compiler
 long_description \
     A free IDL (Interactive Data Language) compatible \
     incremental compiler (i.e. runs IDL programs).
-master_sites                ${github.homepage}/archive:gdl \
-                            http://www.soest.hawaii.edu/pwessel/gshhg:gshhs
+master_sites                ${github.homepage}/archive:gdl 
```

Comment:
Normally the GitHub portgroup sets this properly. You may need to add the line:

`github.tarball_from archive
`

Reply:
wy not just do that? (somebody with knowledge= not me).

Reply:
Maybe I don't have the time or interest to change it and test it (I don't). You are the submitting the PR and are expected to make the changes and do the testing. That is if you want my commit approval. Of course any other committer can commit it without me.

Reply:
@mf2k , I did not mean you...